dio/lib/src/dio/dio_for_browser.dart

```diff
@@ -29,9 +27,31 @@ class DioForBrowser with DioMixin implements Dio {
     String lengthHeader = Headers.contentLengthHeader,
     Object? data,
     Options? options,
-  }) {
-    throw UnsupportedError(
-      'The download method is not available in the Web environment.',
+  }) async {
+    options ??= DioMixin.checkOptions('GET', options);
+
+    options = options.copyWith(responseType: ResponseType.bytes);
+
+    // Set receiveTimeout to 48 hours because `Duration.zero` not work!
+    options = options.copyWith(receiveTimeout: const Duration(hours: 48));
```

Comment:
This doesn't look right.

Reply:
Every time the download started, it timed out for 3 seconds and the download was canceled. I had to use this method.

Reply:
What is the root cause of the behavior? Does it caused by the implementation of Dart or Flutter?